composer.json

```
@@ -45,8 +45,8 @@
         "phpunit/phpunit": "9.5.0",
         "psalm/plugin-phpunit": "0.13.0",
         "squizlabs/php_codesniffer": "3.6.0",
-        "symfony/cache": "^5.2",
-        "symfony/console": "^2.0.5|^3.0|^4.0|^5.0",
+        "symfony/cache": "^5.2|^6.0",
+        "symfony/console": "^2.0.5|^3.0|^4.0|^5.0|^6.0",
```

```suggestion
        "symfony/console": "^3.4|^4.4|^5.2|^6.0",
```
There is no point to support the not maintained versions IMHO.

---

I don't have opinion about this, but this is not the purpose of this PR, and think that should be discussed in a dedicated PR.